src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/InteropExtensions.cs

```diff
@@ -69,9 +69,7 @@ public static IntPtr GetRawFunctionPointerForOpenStaticDelegate(this Delegate de
             if (!del.IsOpenStatic)
                 return IntPtr.Zero;
 
-            RuntimeTypeHandle typeOfFirstParameterIfInstanceDelegate;
-
-            IntPtr funcPtr = del.GetFunctionPointer(out typeOfFirstParameterIfInstanceDelegate, out bool _, out bool _);
+            IntPtr funcPtr = del.GetFunctionPointer(out RuntimeTypeHandle _, out bool _, out bool _);
```

Comment:
Are the types necessary here (e.g. for disambiguation) or could this just have been:
```C#
IntPtr funcPtr = del.GetFunctionPointer(out _, out _, out _);
```
?

Reply:
I believe your simpler version would work (I just kept the syntax to match)